readme.md

```diff
@@ -80,6 +80,10 @@ Exclude certain lambda functions from being wrapped by the plugin. Comma separat
 
 By default, the plugin sends _anonymized_, non-identifying usage statistics to Google Analytics. IOpipe will use this info to prioritize updates and enhancements to the plugin. If you'd like to opt out of this, just set this option.
 
+#### `iopipeHandlerDir` (optional)
```

Comment:
Does this need to be configurable? What's the use case?

Reply:
My opinion is it only needs to be configurable due to a bug in the serverless-offline plugin which is a high-priority support imo for this plugin. After that bug is fixed, we could potentially drop this option.

Reply:
Ok, makes sense. Would it be easier to make this an undocumented option?

Reply:
If it's an opt I believe it should be documented. Thanks!